src/main/java/algorithms/curated170/hard/employeefreetime/EmployeeFreeTimePriorityQueue.java

```diff
@@ -0,0 +1,42 @@
+package algorithms.curated170.hard.employeefreetime;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.PriorityQueue;
+
+public class EmployeeFreeTimePriorityQueue {
+
+    public List<Interval> employeeFreeTime(List<List<Interval>> schedule) {
+        PriorityQueue<Interval> earliestIntervals = new PriorityQueue<Interval>((a, b) -> a.start - b.start);
+        schedule.forEach(e -> earliestIntervals.addAll(e));
+
+        int end = earliestIntervals.poll().end;
+        List<Interval> commonIntervals = new ArrayList<>();
+
+        while (!earliestIntervals.isEmpty()) {
+            Interval interval = earliestIntervals.poll();
+
+            if (interval.start > end && end >= 0) {
```

Comment:
```suggestion
            if (interval.start > end) {
```
Constraint:
`0 <= schedule[i].start < schedule[i].end <= 10^8
`

Reply:
It's a mistake. As in the other file, end is initialized as -1.